packages/now-ruby/gem-install.ts

```diff
@@ -0,0 +1,70 @@
+import { join } from 'path';
+import fetch from 'node-fetch';
+import execa from 'execa';
+import { createWriteStream } from 'fs';
+import { getWriteableDirectory } from '@now/build-utils';
+
+// portable ruby from https://portableruby.com/ for bootstrapping
+// ruby deployment on nodejs8.10. version 2.5.3 matches ruby2.5.
+const url = 'https://portableruby.s3.amazonaws.com/ruby-2.5.3.zip';
+
+// downloads `ruby` and `gem` and returns their absolute paths
+async function downloadPortableRuby(path: string) {
+  console.log('downloading "ruby-2.5.3.zip"...');
+  const res = await fetch(url);
+
+  if (!res.ok || res.status !== 200) {
```

Comment:
```suggestion
  if (res.status !== 200) {
```

You don't need to check both because [ok](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) is 200-299.